sentences/pt/_common.yaml

```diff
@@ -136,6 +142,9 @@ expansion_rules:
   qual: "(que|qual|qual é|quais)"
   temperatura: "{temperature}[°| graus] [{temperature_unit}]"
   zona: "[(o|os|a|as)] {area}"
+  estores: "(cortina|cortinas|cortinado|cortinados|estore|estores|persiana|persianas)"
+  definir_valor: "(coloca|muda|aumenta|sobe|diminui|baixa)"
```

Comment:
we already have `muda` rule. can we use that one instead of adding a new one?

Reply:
Thanks. That make sense. Changed.